### PR TITLE
Update Staking Tutorial and Band Protocol

### PR DESCRIPTION
**Pull request by themacexpert**
**For Staking Tutorial**:
- Update Tutorial and images to show process including getting collator nominator count and inputting nomination count.
- Change Collator Address 2 that no longer existed to an active Collator in Moonbase Alpha
- Update Images and steps for verifying nomination state to suggest enabling "Include Option"

**For Band Protocol**:
- Update formatting of function names for consistent formatting. 

**Other changes** (will separate distinct content into separate PRs in future):
- Embed video for Open Zeppelin Wizard
- Typo in Dapps List Update

### CODE REVIEW

File: integrations/oracles/band-protocol.md [hunk 1]
```diff
@@ -49,15 +49,15 @@ struct ReferenceData {
 }
 ```
 
-The second function, `getReferenceDataBulk`, takes information as data arrays. For example, if we pass in `['BTC','BTC','ETH']` as base and `['USD','ETH','EUR']` as quote, the `ReferenceData`returned array contains the information regarding the following pairs:
+The second function, **getReferenceDataBulk**(*string[]* _bases, *string[]* _quotes), takes information as data arrays. For example, if we pass in `['BTC','BTC','ETH']` as base and `['USD','ETH','EUR']` as quote, the `ReferenceData`returned array contains the information regarding the following pairs:
```
Comment (albertov19) [inline integrations/oracles/band-protocol.md:52]: Here I would leave the function name as it was. I would only change it  in bullets lists and we have to be consistent with that formatting

File: integrations/oracles/band-protocol.md [hunk 2]
```diff
@@ -49,15 +49,15 @@ struct ReferenceData {
...
 
  - `BTC/USD`
  - `BTC/ETH`
  - `ETH/EUR`
 
 ### Example Contract
 
-The following smart contract code provides some simple examples of the `StdReference` contract and the `getReferenceData` function - these are not meant for production. The `IStdReference.sol` interface defines ReferenceData structure and the functions available to make the queries.
+The following smart contract code provides some simple examples of the `StdReference` contract and the **getReferenceData**(*string[]* bases, *string[]* quotes) function - these are not meant for production. The `IStdReference.sol` interface defines ReferenceData structure and the functions available to make the queries.
```
Comment (albertov19) [inline integrations/oracles/band-protocol.md:60]: Same as before

File: integrations/oracles/band-protocol.md [hunk 3]
```diff
@@ -181,13 +181,13 @@ This will create an instance of the demo contract that you can interact with. Us
 
 ## BandChain.js Javascript Helper Library
 
-The helper library also supports a similar `getReferenceData` function. To get started, the library needs to be installed:
+The helper library also supports a similar **getReferenceData**(*string[]* base/quote) function. To get started, the library needs to be installed:
```
Comment (albertov19) [inline integrations/oracles/band-protocol.md:184]: Same as before

File: integrations/oracles/band-protocol.md [hunk 4]
```diff
@@ -181,13 +181,13 @@ This will create an instance of the demo contract that you can interact with. Us
...
 
 ```
 npm install @bandprotocol/bandchain.js
 ```
 
-The library provides a constructor function that requires an endpoint to point to. This returns an instance that then enables all the necessary methods, such as the `getReferenceData` function.  When querying for information, the function accepts an array where each element is the _base/quote_ pair needed. For example:
+The library provides a constructor function that requires an endpoint to point to. This returns an instance that then enables all the necessary methods, such as the **getReferenceData**(*string[]* base/quote) function.  When querying for information, the function accepts an array where each element is the _base/quote_ pair needed. For example:
```
Comment (albertov19) [inline integrations/oracles/band-protocol.md:190]: Same here

File: integrations/oracles/band-protocol.md [hunk 5]
```diff
@@ -218,7 +218,7 @@ Where `lastUpdatedBase` and `lastUpdatedQuote` are the last time when the base a
 
 ### Example Usage
 
-The following Javascript script provides a simple example of the `getReferenceData` function.
+The following Javascript script provides a simple example of the **getReferenceData**(*string[]* base/quote) function.
```
Comment (albertov19) [inline integrations/oracles/band-protocol.md:221]: And here

File: staking/stake.md [hunk 6]
```diff
@@ -63,9 +63,24 @@ Each extrinsic provides a different response:
...
+## Get the Collator Nominator Count
+
+First, you need to get the `collator_nominator_count` as you'll need to submit this parameter in a later transaction. To do so, you'll have to run the following JavaScript code snippet from within [PolkadotJS](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fwss.testnet.moonbeam.network#/js):
+
+
+
+```js
+// Simple script to get collator_nominator_count
+// Remember to replace collatorAccount with the address of desired collator.
+const collatorAccount = '0x4c5A56ed5A4FF7B09aA86560AfD7d383F4831Cce';
```
Comment (albertov19) [inline staking/stake.md:75]: Replace the actual collator address with COLLATOR_ADDRESS

File: staking/stake.md [hunk 7]
```diff
@@ -87,9 +102,14 @@ To nominate a collator, provide the following information:
  3. Choose the extrinsic method to use for the transaction. This will determine the fields that need to fill in the following steps. In this case, it is the `nominate` extrinsic
  4. Set the collator's address you want to nominate. In this case, it is set to `{{ networks.moonbase.staking.collators.address1 }}`
  5. Set the number of tokens you want to stake
- 6. Click the "Submit Transaction" button and sign the transaction
+ 6. Input the `collator_nominator_count` you retrieved above from the JavaScript console
```
Comment (albertov19) [inline staking/stake.md:105]: Link to the section

File: staking/stake.md [hunk 8]
```diff
@@ -87,9 +102,14 @@ To nominate a collator, provide the following information:
...
+ 7. Input the number of previous nominations you have made from your selected account. This would be 0 if you haven't yet nominated a collator. If you are uncertain of how many nominations you have already made, estimate a maximum number
```
Comment (albertov19) [inline staking/stake.md:106]: Is it possible to get a value using javascript?

**Review by albertov19 — CHANGES_REQUESTED** [review]

File: staking/stake.md (hunk 8 above)
Comment (themacexpert) [reply staking/stake.md]: Yes! Updated with the JS query.

**Review by albertov19 — CHANGES_REQUESTED** [review]
Comments already sent :)

**Comment by themacexpert** [comment]
Ready for review 👍  @albertov19

**Review by albertov19 — APPROVED** [review]

**Pull request merged**